cmds/core/du/du.go

```diff
@@ -47,7 +47,10 @@ func command(stdout io.Writer, args []string) *cmd {
 	f.BoolVar(&c.followCMDSymLinks, "H", false, "follow symlink form [file...]")
 	f.BoolVar(&c.followSymlinks, "L", false, "follow all symlinks")
 
-	f.Parse(unixflag.ArgsToGoArgs(args[1:]))
+	if err := f.Parse(unixflag.ArgsToGoArgs(args[1:])); err != nil {
+		fmt.Fprintf(os.Stderr, "error parsing flags: %v\n", err)
```

Comment:
would log.Fatal work here?
also does f.Parse exit on its own by default? Then this err check is unnecessary anyway. f.Parse will exit .
This may be another case where golangci-lint does not understand what's really going on.